### PR TITLE
IRIS-4169 | OSN feature flag does not affect wikia-nav

### PR DESCRIPTION
**Pull request by slayful**
## Links

* http://wikia-inc.atlassian.net/browse/IRIS-4169

### CODE REVIEW

File: front/main/app/templates/components/wikia-nav.hbs [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+					<figure class="wikia-nav__avatar">
+						<img src={{currentUser.avatarPath}} alt="{{currentUser.name}}">
+					</figure>
+					{{i18n 'app.user-menu-profile'}}
+				</a>
+				<a {{action 'trackClick' 'side-nav' 'open-logout' preventDefault=false}}
+						href="{{unbound logoutLink}}"
+						class="wikia-nav--external wikia-nav--logout">
+					{{i18n 'app.user-menu-log-out'}}
+				</a>
```
Comment (slayful) [inline front/main/app/templates/components/wikia-nav.hbs]: Basically copy-pasted previous behaviour and wrapped it in the feature toggle `if-else`.
The `else` branch will be removed when OSN are considered stable on production.

File: front/main/app/components/wikia-nav.js [hunk 2]
```diff
@@ -7,14 +7,28 @@ import {track, trackActions} from 'common/utils/track';
...
 	{
 		classNames: ['wikia-nav'],
 		classNameBindings: ['model.inRoot:wikia-nav--in-root'],
 		currentUser: inject.service(),
 		notifications: inject.service(),
 		isUserAuthenticated: computed.oneWay('currentUser.isAuthenticated'),
+		enableOnSiteNotifications: Ember.get(Mercury, 'wiki.enableOnSiteNotifications'),
+
+		/** TODO: Remove with the feature flag IRIS-4170 */
+		logoutLink: M.buildUrl({
```
Comment (latata) [inline front/main/app/components/wikia-nav.js]: We currently have `/logout` URL. I think we should currently use this one.

File: front/main/app/templates/components/wikia-nav.hbs [hunk 3]
```diff
@@ -1,17 +1,33 @@
...
+						{{#if unreadCount}}
+							<span class="notifications-unread-mark">{{unreadCount}}</span>
+						{{/if}}
+						<img src={{currentUser.avatarPath}} alt="{{currentUser.name}}">
+					</figure>
+					{{currentUser.name}}
+					{{svg 'wds-icons-menu-control' viewBox='0 0 24 24' class='icon menu-chevron'}}
+				</a>
+			{{else}}
+				<!--TODO: Remove with the feature flag IRIS-4170 -->
```
Comment (kvas-damian) [inline front/main/app/templates/components/wikia-nav.hbs]: Please use HBS comments instead. HTML comments are visible in view source  ;)

File: front/main/app/templates/components/wikia-nav.hbs [hunk 4]
```diff
@@ -28,7 +44,7 @@
 {{/if}}
 
 {{nav-menu
-	items=model.items
-	onClickHandler=(action 'onClick')
-	onChangeItemHandler=(action 'goToSubNav')
+		items=model.items
```
Comment (latata) [inline front/main/app/templates/components/wikia-nav.hbs]: I think we should have single indentation here.

File: front/main/app/components/wikia-nav.js (hunk 2 above)
Comment (slayful) [inline front/main/app/components/wikia-nav.js]: I'd rather not focus on the behaviour that was there and will only be removed after On Site Notifications are turned on.

**Pull request merged**